members/federal_us_veterans_sec_robert_wilkie.yaml

```diff
@@ -32,8 +33,8 @@ contact_form:
     - click_on:
       - value: Inquiry
         selector: button[id*='Category_Button']
-      - value: Question
-        selector: "#ygtvlabelel26"
+      - value: Suggestion
```

Comment:
Do we still want to make this change if it doesn't fix the submission process?

(I have no objections, it seems more appropriate for advocacy in any case.)

Reply:
Yeah, I think it's worth updating at least.